compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -120,6 +143,8 @@ struct ConvertToROCDLPass : public ConvertToROCDLBase<ConvertToROCDLPass> {
       populateFuncToLLVMConversionPatterns(converter, llvmPatterns);
       cf::populateControlFlowToLLVMConversionPatterns(converter, llvmPatterns);
       arith::populateArithToLLVMConversionPatterns(converter, llvmPatterns);
+      FailureOr<amdgpu::Chipset> maybeChipset = amdgpu::Chipset::parse("gfx90a");
```

Comment:
Can you add a brief comment explaining the choice of `gfx90a`?

Reply:
I parameterized it